packages/spacecat-shared-data-access/src/models/import-job/import-job.schema.js

```diff
@@ -35,6 +35,16 @@ const ImportOptionTypeValidator = {
       throw new Error(`Invalid value for ${ImportJob.ImportOptions.PAGE_LOAD_TIMEOUT}: ${value}`);
     }
   },
+  [ImportJob.ImportOptions.TYPE]: (value) => {
+    if (!Object.values(ImportJob.ImportOptionTypes).includes(value)) {
+      throw new Error(`Invalid value for ${ImportJob.ImportOptions.TYPE}: ${value}`);
+    }
+  },
+  [ImportJob.ImportOptions.DATA]: (value) => {
+    if (!isObject(value)) {
```

Comment:
is the value nullable? if so i'd check for `if (value && !isNonEmptyObject(value))` - would not necessarily store an empty object for nothing.